js/rendererjs/loadingScreen.js

```diff
@@ -36,10 +37,17 @@ const checkSiaPath = () => new Promise((resolve, reject) => {
 })
 // startUI starts a Sia UI instance using the given welcome message.
 // calls initUI() after displaying a welcome message.
-const startUI = (welcomeMsg, initUI) => {
+const startUI = async (welcomeMsg, initUI) => {
 	// Display a welcome message, then initialize the ui
 	overlayText.innerHTML = welcomeMsg
 
+	// construct the autoupdater component and check for updates every 30 minutes.
+	const checkUpdate = async () => ReactDOM.render(await Update(currentVersion), document.getElementById('update-status-container'))
+	setInterval(async () => {
+		await checkUpdate()
+	}, 18000000)
+	await checkUpdate()
```

Comment:
it does block.  If you don't have an internet connection the UI doesnt load, fixing now...